examples/covariance/plot_outlier_detection.py

```diff
@@ -0,0 +1,129 @@
+"""
```

Comment:
Is it a new file?

Reply:
This example was removed in PR #10700 (merged yesterday) in favor of plot_anomaly_comparison.py. We don’t need it anymore.